Frontend/ui-driver/src/MerchantConfigs/Types.purs

```diff
@@ -126,4 +128,9 @@ type MapConfig = {
 type WaitTimeConfig = {
   enableWaitTime :: Boolean,
   thresholdDist :: Number
+}
+
+type LiveStatsDashboard = {
+  showLiveStatsDashboard :: Boolean,
```

Comment:
this also can be called as just `show`, `visible` as this is within `LiveStatsDashboard` config